modules/virt-about-deploying-templates-private-namespace.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-about-deploying-templates-private-namespace.adoc
+
+[id="virt-about-deploying-templates-private-namespace_{context}"]
+= About deploying templates to a private namespace
+
+To add templates to a private namespace for private use, edit the `hco` custom resource (CR) and specify a private namespace. Then, add the Red Hat templates to that private namespace.
```

Comment:
We should avoid using `private` unless absolutely necesssary in the documentation. Private is up to the admin, changing the namespace will only cause them to be hidden from the UI.

Reply:
* will only cause them to be hidden from the UI. [after deleted from default namespace]

Reply:
Updated the content by removing "private".